src/Auth/Views/Account/Logout.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@model LogoutViewModel
+
+<div class="logout-page">
+    <div class="page-header">
+        <h1>Logout</h1>
+    </div>
+
+    <div class="row">
+        <div class="col-sm-6">
+            <p>Would you like to logout of IdentityServer?</p>
```

Comment:
I think this is impossible because the token does not hold the client name with which it has been authenticated. I am 90% sure. It might be possible on login but not log out. I might be wrong

Reply:
In that case... well, what does this button actually _do_? Does it log you out only of our platform, or out of external IdPs as well?

Reply:
Just "our platform"
```

      // delete local authentication cookie
       await _signInManager.SignOutAsync();
```

Reply:
We must obtain the client name somehow then...

Reply:
Is it possible to access either the settings db or the settings file there?

Reply:
I'd like this to show "Would you like to log out of $SiteName" - doesn't matter where that comes from, but presumably it'll be a configurable item so that folks running their own can rename the site. Can we access those site settings here?

Reply:
Yes, I believe this should come from a method or property defined in the Infrastructure project.

Reply:
I did this: https://github.com/misha130/core/commit/b8be380a9481ec02c947881b60e67153eff7ea59
Its on another branch cause I don't like it much.
What do you think

Reply:
Looks good to me, @misha130

Reply:
Then I merged. I think thats it.